cloud/create-databases.yml

```diff
@@ -0,0 +1,19 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+    - "{{ local_secret_dir }}/{{ cluster }}/cluster-secrets.yml"
+    - "{{ local_secret_dir }}/cloud/cloud-secrets.yml"
+  vars:
+    choose_database_option: create
```

Comment:
Please rename this to `database_operation`

Reply:
Your solution repeat the `database_state` variable.
```
- {role: cloud_sql, cluster_type: control, database_state: present}
- {role: cloud_sql, cluster_type: storage, database_state: present}